sdk/cognitivelanguage/ai-language-conversations/src/conversationAnalysisClient.ts

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+import * as coreClient from "@azure/core-client";
+import * as coreRestPipeline from "@azure/core-rest-pipeline";
+import {
+    bearerTokenAuthenticationPolicy,
+  PipelineRequest,
+  PipelineResponse,
+  SendRequest
+} from "@azure/core-rest-pipeline";
+import { PollerLike, PollOperationState, LroEngine } from "@azure/core-lro";
+import { LroImpl } from "./generated/lroImpl";
+import * as Parameters from "./generated/models/parameters";
+import * as Mappers from "./generated/models/mappers";
+import {
+  ConversationAnalysisClientOptionalParams,
+  AnalyzeConversationTaskUnion,
+  AnalyzeConversationOptionalParams,
+  AnalyzeConversationResponse,
+  AnalyzeConversationJobsInput,
+  ConversationAnalysisOptionalParams,
+  ConversationAnalysisResponse
+} from "./generated/models";
+import { conversationAnalysisAzureKeyCredentialPolicy } from "./azureKeyCredentialPolicy";
+import { DEFAULT_COGNITIVE_SCOPE } from "./constants";
+import { isTokenCredential, KeyCredential, TokenCredential } from "@azure/core-auth";
+
+/** @internal */
+export class ConversationAnalysisClient extends coreClient.ServiceClient {
```

Comment:
could you please create a fresh client that doesn't extend anything and then use the generated client internally? you can see an example of how to do that in ai-language-text as well.

Reply:
Should I also use a tracing client?

Reply:
Yes please